src/plugins/discourse/update_discourse_api_snapshots.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -eu
+
+snapshots_dir=src/plugins/discourse/snapshots
+test_instance_url="https://sourcecred-test.discourse.group"
+test_instance_username="credbot"
+
+if [ -z "${DISCOURSE_TEST_API_KEY:-}" ]; then
+  printf >&2 'Please set the DISCOURSE_TEST_API_KEY environment variable.\n'
+  printf >&2 'Contact the SourceCred maintainers to get the key.\n'
+  return 1
+fi
+
+toplevel="$(git -C "$(dirname "$0")" rev-parse --show-toplevel)"
+cd "${toplevel}"
+
+fetch() {
+  url="${test_instance_url}${1}"
+  filename=$(echo -n "${url}" | base64 -w 0 | tr -d '=' | tr '/+' '_-')
```

Comment:
Recommend quoting `filename="$(…)"`. It’s not technically necessary in
variable assignments, but if you quote then it’s _clearly_ correct
rather than _merely_ correct.